spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -37,38 +37,55 @@ val NON_LICENSE_FILENAMES = listOf(
 private fun ByteArray.toHexString(): String = joinToString("") { String.format("%02x", it) }
 
 /**
- * Calculate the [SPDX package verification code][1] for a list of [known SHA1s][sha1sums] of files.
+ * Calculate the [SPDX package verification code][1] for a list of [known SHA1s][sha1sums] of files and [excludes].
  *
  * [1]: https://spdx.org/spdx_specification_2_0_html#h.2p2csry
  */
 @JvmName("calculatePackageVerificationCodeForStrings")
-fun calculatePackageVerificationCode(sha1sums: List<String>): String =
-    sha1sums.sorted().fold(MessageDigest.getInstance("SHA-1")) { digest, sha1sum ->
+fun calculatePackageVerificationCode(sha1sums: List<String>, excludes: List<String> = emptyList()): String {
+    val sha1sum = sha1sums.sorted().fold(MessageDigest.getInstance("SHA-1")) { digest, sha1sum ->
         digest.apply { update(sha1sum.toByteArray()) }
     }.digest().toHexString()
 
+    if (excludes.isEmpty()) {
+        return sha1sum
+    } else {
+        return "$sha1sum (excludes: ${excludes.joinToString()})"
```

Comment:
Nit: Could move the `return`s out of `if`.

Reply:
Done.